Automattic/jetpack
# PHPCS rule to prefer implode over join

## Proposed changes:

This adds a phpcs rule to enforce using `implode` instead of `join`.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Check out this branch.
* Make an update that uses the `join` function and try to commit it.
* Your code should automatically be updated to use `implode` instead of `join`.
